docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -99,17 +99,13 @@
       provided to facilitate filtering records of interest from the HPCC
       Cluster.</para>
 
-      <para>The git repository includes two examples under the
-      Examples/src/main/scala folder. The examples
-      (<emphasis>org.hpccsystems.spark_examples.Dataframe_Iris_LR</emphasis>
-      and <emphasis>org.hpccsystems.spark_examples.Iris_LR</emphasis>) are
-      Scala Objects with a main() function. Both examples use the classic Iris
-      dataset. The dataset can be obtained from a variety of sources,
-      including the HPCC-Systems/ecl-ml repository. IrisDs.ecl (can be found
-      under the ML/Tests/Explanatory folder: <ulink
-      url="https://github.com/hpcc-systems/Spark-HPCC/blob/master/Examples/src/main/ecl/IrisDS.ecl">https://github.com/hpcc-systems/Spark-HPCC/blob/master/Examples/src/main/ecl/IrisDS.ecl</ulink>)
-      can be executed to generate the Iris dataset in HPCC. A walk-through of
-      the examples is provided in the Examples section.</para>
+      <para>The Github repository includes examples in the Examples folder.
```

Comment:
@jpmcmu Greg is removing broken links in the doc, I believe the samples discussed here no longer exist. I think this section should be removed, please confirm. (the existing sample can be added via hpcc4j-679)